index.html

```diff
@@ -9,7 +9,7 @@
     <title>Day 01</title>
   </head>
   <body>
-    <!-- 코드 작성 하세요 -->
-    <!-- Don't delete this link:fontawsome -->
+    <h1>hello first html!</h1>
+    <p>kdskdskkdsksldkdskdskl</p>
```

Comment:
p태그는 문단 태그입니다. span 태그로 고치셔야합니다.
```suggestion
    <span>kdskdskkdsksldkdskdskl</span>
```